lib/streamlit/commands/navigation.py

```diff
@@ -126,15 +126,47 @@ def navigation(
     you pass to ``streamlit run``. Your entrypoint file manages your app's
     navigation and serves as a router between pages.
 
+    **Example 1: Use a callable or Python file as a page**
+
     You can declare pages from callables or file paths.
 
+    ``Page_1.py`` (in the same directory as your entrypoint file):
+
+    >>> import streamlit as st
+    >>>
+    >>> st.title("Page 1")
+
+    Your entrypoint file:
+
     >>> import streamlit as st
-    >>> from page_functions import page1
     >>>
-    >>> pg = st.navigation([st.Page(page1), st.Page("page2.py")])
+    >>> def Page_2():
```

Comment:
https://github.com/streamlit/streamlit/pull/9765

Reply:
Off. I was trying to avoid adding the page title manually and was relying on the function name and file name to yield "Page 1" and "Page 2" in the navigation. If you think it's better, I can downgrade both to "page 1" and "page 2" and not highlight using the callable/file name for anything pretty.

Reply:
noo it's fine like this and no strong opinion, just if anyone copy & pastes it into their IDE, their linter might complain 🙂